README.md

```diff
@@ -1,242 +1,6 @@
 # OONI backend
 
-Welcome. This document describes the architecture of the main components of the
-OONI infrastructure.
+Welcome to the OONI backend.
```

Comment:
```suggestion
Welcome to the OONI backend!
```

Does the referenced documentation include information on how to build and tests?

Reply:
I'm not sure what you mean by that. In any case it's the latest documentation we have about anything backend related. If it's not in there it should be added, but that's orthogonal to this PR I think.

Reply:
I mean that I suggest adding a `!` at the end of the sentence because it's more welcoming!

I also suggest having documentation about building and testing withing the backend itself rather than externally.

Reply:
Yeah that would be nice, however I focused here on removing all the docs that are wrong to prevent confusion. While it would be nice to have that in the readme, that's I think out of scope for this PR.